server/src/tasks/schema.js

```diff
@@ -43,12 +43,25 @@ const taskResolvers = {
 
   Mutation: {
     createTask: async (obj, args, context, info) => {
+      const message = {
```

Comment:
-1000000000000000000000000000.
Can this be abstracted so we will not obfuscate tasks/schema with some push specific implementation. 
We currently use publish method. Let's make publish method use push as well. 
Let's keep this modular (see how subscriptions were made)

Reply:
@wtrocki good idea. i'll do that, thanks